### DISCUSSION TITLE
Is it possible to hide slash commands from other people?

### DISCUSSION THREAD

**Discussion by IIDarkNessYT** [Rapptz/discord.py#8332] [Q&A] [answered]
Hi all! I'm making slash commands and I want some to be visible only to me or to users with a specific ID, is it possible?

**Comment by SebbyLaw** [comment, marked as answer]
No. Only the server owner or admins can do this through the "integrations" tab in server settings. So you'd have to be the owner or admin in every server your bot is in.

**Reply by IIDarkNessYT** [reply]
Thanks
